torchvision/models/inception.py

```diff
@@ -63,28 +63,45 @@ def inception_v3(pretrained=False, progress=True, **kwargs):
 
 class Inception3(nn.Module):
 
-    def __init__(self, num_classes=1000, aux_logits=True, transform_input=False):
+    def __init__(self, num_classes=1000, aux_logits=True, transform_input=False,
+            basic_conv2d=None, inception_a=None, inception_b=None,
```

Comment:
This is ok but a lot of different blocks.
Can you maybe do something like
```python
inception_blocks=None)
...
if inception_blocks is None:
    inception_blocks = [BasicConv2d, InceptionA, InceptionB, InceptionC, ...]

assert len(inception_blocks) == 7
```
and use `inception_blocks[0]` etc in the code?